aws/resource_aws_vpn_connection_test.go

```diff
@@ -569,18 +766,88 @@ resource "aws_customer_gateway" "customer_gateway" {
 }
 
 resource "aws_vpn_connection" "test" {
-  vpn_gateway_id      = "${aws_vpn_gateway.vpn_gateway.id}"
-  customer_gateway_id = "${aws_customer_gateway.customer_gateway.id}"
+  vpn_gateway_id      = aws_vpn_gateway.vpn_gateway.id
+  customer_gateway_id = aws_customer_gateway.customer_gateway.id
   type                = "ipsec.1"
   static_routes_only  = false
 
-  tunnel1_inside_cidr   = "%s"
-  tunnel1_preshared_key = "%s"
-
-  tunnel2_inside_cidr   = "%s"
-  tunnel2_preshared_key = "%s"
-}
-`, rBgpAsn, tunnelCidr, psk, tunnelCidr2, psk2)
+  local_ipv4_network_cidr  = "%s"
+  remote_ipv4_network_cidr = "%s"
+
+  tunnel1_inside_cidr                  = "%s"
```

Comment:
Specifying which argument to use will make this easier to troubleshoot later. eg
`tunnel1_inside_cidr                  = %[3]q`

Reply:
Done